html-css/week2/index.html

```diff
@@ -0,0 +1,168 @@
+                    </div>
+                    <div class="info-text">
+                        <h1>Logotypes on the web</h1>
+                        <div class="info-icons">
+                                <img src="./img/icon_1.svg" alt="">
+                                <img src="./img/icon_2.svg" alt="">
+                                <img src="./img/icon_3.svg" alt="">
+                                <img src="./img/icon_4.svg" alt="">
+                                <img src="./img/icon_4.svg" alt="">
+                                <img src="./img/icon_3.svg" alt="">
+                                <img src="./img/icon_2.svg" alt="">
+                                <img src="./img/icon_1.svg" alt="">
+                                <img src="./img/icon_2.svg" alt="">
+                                <img src="./img/icon_1.svg" alt="">
+                                <img src="./img/icon_4.svg" alt="">
+                                <img src="./img/icon_3.svg" alt="">
+                        </div> 
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+
+    <footer id="footer" class="footer">
+        <div class="row">
+            <div>
+                <h3>Find us</h3>
+                <iframe
+                    src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2249.4924506324096!2d12.606524859370031!3d55.680425669897936!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x465253256f012ff3%3A0xe812f0cada10194c!2sTalent%20Garden%20Rainmaking!5e0!3m2!1suk!2sdk!4v1664291227587!5m2!1suk!2sdk"
+                    width="200" height="200" style="border:0;" allowfullscreen="" loading="lazy"
+                    referrerpolicy="no-referrer-when-downgrade"></iframe>
+                <ul>
+                    <li>
+                        Hack Your Future
+                    </li>
+                    <li>
+                        Copenhagen
+                    </li>
+                    <li>
+                        M-Th: 7am - 4pm
+                    </li>
+                    <li>
+                        Fr-Sat: 9am - 8pm
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <h3>About this page</h3>
+                <p>This is a footer of the page for homework 02</p>
+            </div>
+            <div>
+                <h3>Archives</h3>
+                <ul>
+                    <li>
+                        <a href="">March 2016</a> 
+                    </li>
+                    <li>
+                        <a href="">February 2016</a>   
+                    </li>
+                    <li>
+                        <a href="">January 2016</a>   
+                    </li>
+                </ul>
+            </div>
+            <div>
+                <h3>Navigation</h3>
+                <ul>
+                    <li>
+                        <a href="https://imgur.com/Lv2IGgf">Home</a>
```

Comment:
use `target="_blank"` with external links of `a` tag, we talked about it in our last class